### PR TITLE
Runthrough tweaks

### PR DESCRIPTION
**Pull request by rustydb**
#### Issue Type

- Bugfix Pull Request
- Docs Pull Request

Includes various amendments from my last runthrough.

- Doc fixups
- Usage of the RPM GPG key
- New playbook calls for setting up the management VM

### Prerequisites

- [x] I have included documentation in my PR (or it is not required)
- [x] I tested this on internal system (if yes, please include results or a description of the test)
- [ ] I tested this on a vshasta system (if yes, please include results or a description of the test)

### CODE REVIEW

**Review by manderson-hpe — APPROVED** [review]

File: modules/installation/pages/2-pre-installation/1-usb-creation.adoc [hunk 1]
```diff
@@ -117,11 +118,16 @@ All the images should return with `OK`. If any fail, please stop and open a bug
 +
 NOTE: `crucible` is undergoing source code changes and will have packages for `yum` and `debian` package managers in the near future.
 +
-* SUSE
+* YUM RPM (SUSE/RedHat)
```
Comment (jpdavis-prof) [inline modules/installation/pages/2-pre-installation/1-usb-creation.adoc]: This section feels weird. Did you mean to have **YUM RPM (SUSE/RedHat)**

**Comment by jpdavis-prof** [comment]
nit: In **1-usb-creation.adoc**, section **MacOS and Windows** it should say something like:

Creating a USB **drive** from MacOS and Windows is feasible using the help of a few third-party tools.

**Comment by jpdavis-prof** [comment]
This guide walks the user through setting up a bootable USB **drive** for bootstrapping the Fawkes cluster.

**Comment by jpdavis-prof** [comment]
The following steps will create a USB **drive** from a Linux OS.

File: modules/installation/pages/2-pre-installation/1-usb-creation.adoc (hunk 1 above)
Comment (rustydb) [reply modules/installation/pages/2-pre-installation/1-usb-creation.adoc]: I meant just any RPM, so I could ditch the "YUM" bit.

**Comment by jpdavis-prof** [comment]
Does this work with our Antora docs? https://github.com/Cray-HPE/fawkes-docs/blob/5e2251bdc9a7063062428017af34687b787f8683/modules/installation/pages/2-pre-installation/1-usb-creation.adoc#steps

All the images should return with OK. If any fail, please stop and open a bug by using the bug opening tab on the bottom right of this screen.

**Review by jpdavis-prof — APPROVED** [review]

**Comment by rustydb** [comment]
> Does this work with our Antora docs? https://github.com/Cray-HPE/fawkes-docs/blob/5e2251bdc9a7063062428017af34687b787f8683/modules/installation/pages/2-pre-installation/1-usb-creation.adoc#steps
> 
> All the images should return with OK. If any fail, please stop and open a bug by using the bug opening tab on the bottom right of this screen.

Thanks for pointing this out, we had to fix this button. It's now showing up.

- https://github.com/Cray-HPE/antora-ui/pull/18
- https://github.com/Cray-HPE/fawkes/pull/40



image

**Pull request merged**